src/pages/Thanks/ThanksPage.vue

```diff
@@ -111,8 +112,13 @@ export default {
 				...data.my.userAccount,
 				teams: data.my.teams.values.map(value => value.team)
 			};
-			this.receipt = data.shop.receipt;
-			this.loans = data.shop.receipt.items.values
+
+			// The default empty object and the v-if will prevent the
+			// receipt from rendering in the rare cases this query fails.
+			// But it will not throw a server error.
+			this.receipt = _get(data, 'shop.receipt', {});
+			const loansResponse = _get(this.receipt, 'items.values', []);
+			this.loans = loansResponse
 				.filter(item => item.basketItemType === 'loan_reservation')
 				.map(item => item.loan);
 		}
```

Comment:
@eddieferrer could we please console an error if the receipt and/or the lender is missing at the end of this result. For the lender we can prob just key off the presence or lack thereof `userAccount`.